### ISSUE TITLE
Download release Windows appveyor artifact

### ISSUE THREAD

**Issue by KyleGass** [kennyfrc/trainingdata-tool#5]
Greetings! I try to download the tainingdata-tool like discribed at the lc0 Homepage (https://lczero.org/dev/wiki/training-and-testing-a-net-on-google-colab-beginner-friendly-guide/). They write, there should be a prebuilt windows binary available, but also the Link (For Windows, you may use this appveyor artifact) directs me to an empy location. "The build job does not contain any artifacts." 
Sorry if its a dumb question, but is there a way to get the trainingdata-tool binarys anymore for Windows(10)?

**Comment by KyleGass** [comment]
Found an actual valid download on the LC0 Discord for anyone who needs it: 
https://cdn.discordapp.com/attachments/427066771627966466/787663547546861568/trainingdata-tool.exe

For anyone following this LC0 guide: Training and Testing a Net on Google Colab: Beginner Friendly Guide
(https://lczero.org/dev/wiki/training-and-testing-a-net-on-google-colab-beginner-friendly-guide/)

I later also needed to create a proto folder in lczero-training-master\tf and put in this file:
https://cdn.discordapp.com/attachments/427066771627966466/822101986597797918/chunk_pb2.py
and this file: 
https://cdn.discordapp.com/attachments/427066771627966466/822101992587395142/net_pb2.py
(and also created a tmp folder with an empty mymodel4.txt file in that folder)

I also had to edit the example.yaml file in the lczero-training-master\tf\configs folder

%YAML 1.2
---
name: t1-128x10b
gpu: 0
dataset:
  num_chunks: 16865
  train_ratio: 0.95
  input_train: /content/gdrive/My Drive/lc0/Training Data/*/
  input_test: /content/gdrive/My Drive/lc0/Training Data/*/
training:
  batch_size: 512
  test_steps: 2000
  train_avg_report_steps: 1
  total_steps: 100000
  checkpoint_steps: 1000
  warmup_steps: 250
  shuffle_size: 524288
  lr_values:
    - 0.02
    - 0.002
    - 0.0005
  lr_boundaries:
    - 60000
    - 80000
  policy_loss_weight: 1
  value_loss_weight: 1
  moves_left_loss_weight: .01
  path: /content/gdrive/My Drive/lc0/
model:
  filters: 128
  residual_blocks: 10
  se_ratio: 8
...

I Created a google drive folder: lc0 put in the lczero-training-master folder, Training Data folder, trainingdata-tool.exe & pgn-extract.exe and the input.pgn as described in https://lczero.org/dev/wiki/training-and-testing-a-net-on-google-colab-beginner-friendly-guide/

with google colab created a notebook & with this commands it finally worked:
from google.colab import drive
drive.mount('/content/gdrive', force_remount=True)
!chmod -R 777 "/content/gdrive/My Drive/lc0/"
!"/content/gdrive/My Drive/lc0/lczero-training-master/tf/train.py" --cfg "/content/gdrive/My Drive/lc0/lczero-training-master/tf/configs/example.yaml" --output "/content/gdrive/My Drive/lc0/lczero-training-master/tf/tmp/mymodel4.txt"
